source/assets/js/components/fef-modal.js

```diff
@@ -156,7 +172,7 @@ export class FefModal {
     setFocus($element) {
         $element.attr('tabindex', -1).on('blur focusout', () => {
             $element.removeAttr('tabindex');
-        }).focus();
+        }).get(0).focus({preventScroll: true});
```

Comment:
geht `.focus()` auf einem DomNode? Dachte das müsste immer auf einem jQuery-Element passieren

Reply:
setFocus gibt's ja schon im FlyingFocus.js (https://github.com/mmz-srf/srf-frontend-framework/blob/master/source/assets/js/components/fef-flying-focus.js#L189) - dort haben wir das `get(0)` nicht, wenn das hier besser ist, vielleicht auch gerne dort anpassen und dann das setFocus vom FF verwenden?

Reply:
es gibt ein jquery focus() und ein native focus(). das native focus() erlaubt es im gegensatz zu dem von jquery, das scrollen zum fokussierten element zu verhindern (preventScroll), was ich in diesem falle wollte, weil es zu komischen scroll-effekten kam. evtl. fliegt das aber wieder raus, wenn das design das anders will.

und der focus() ist hier möglich, weil du auf dem element einen tabindex setzt. damit wird es "focusable".

Reply:
TIL! https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus

Hab immer nur das von jQuery benutzt 🙈 Aber gut zu wissen!